### DISCUSSION TITLE
HiROB - A toy robot teaching kids code

### DISCUSSION THREAD

**Discussion by husamettinarabaci** [Ubn-Jr/hirob-docs#2] [Q&A]
Hello everyone,

**Comment by Sddilora** [comment]
Hi 🤖

**Comment by Maraza25** [comment]
How do we register users, with a serial code or some other format?

**Reply by Sddilora** [reply]
Actually we may use mail and password for authentication, and after user signed in with correct infos we will send jwt for session. We have meeting on this Sunday , after we discuss that with our team we will let you know.

**Comment by Talha-Dogan** [comment]
Hello everyone, I'm @Talha-Dogan, the project manager of [Hirob](https://github.com/orgs/Ubn-Jr/projects/4). We are planning to create a robot that teaches children by developing a block-based platform. The general aim of this robot is to encourage children to think creatively and problem-solve. Our application will enable children to learn by observing the robot's movements after running the code, providing real-life practical learning experiences.

In the development of this robot, we would like every member of the Hirob project to express their opinions on every aspect of the robot.

## Discussion Topics:

- Hirob Project Overview
- Motion Mechanisms (Wheel Selection)
- Robot Design
- Map
- Mobile Application Features

## Sample discussion questions:
- Should Hirob be controlled through a keypad on the robot or via a mobile application?
- Should Hirob have tracks or wheels for its movement mechanism?
- Should the map be a scanned architectural map through the mobile app or a ready-to-use map system within the app?
- Should the external casing of Hirob be made of plastic or wood?
- Is the map dynamic or in a static puzzle-like format?
- External casing design

Feel free to share your thoughts on any topic that comes to mind. 🤖
